backend/src/app.ts

```diff
@@ -22,11 +26,17 @@ app.use(
   })
 );
 
+app.use(passport.initialize());
+app.use(passport.session());
+
 app.use(logger());
 
 // Exposed for a TypeForm webhook
 app.use(hookRoutes);
 
+app.use(usersRoute);
+app.use(authRoutes.routes());
```

Comment:
Can we change the way routes are exported in `authRoutes` to export calling `.routes()` for consistency with outer routes we have?